<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapVisualizationClass</class>
 <widget class="QWidget" name="MapVisualizationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MapVisualization</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="MapRenderer" name="mapRenderer" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Reset Image</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Mode</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QRadioButton" name="voronoiButton">
           <property name="text">
            <string>Voronoi</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="shepardButton">
           <property name="text">
            <string>Shepard</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MapRenderer</class>
   <extends>QWidget</extends>
   <header>maprenderer.h</header>
   <container>1</container>
   <slots>
    <slot>reset()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>mapRenderer</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setVoronoi()</slot>
  <slot>setShepard()</slot>
 </slots>
</ui>
